.github/workflows/pr-title.yml

```diff
@@ -41,6 +41,11 @@ jobs:
     # when called as a reusable workflow (the worker is never started, and the
     # queued job just hangs there).
     steps:
+      - name: Fail if the GITHUB_PAT input is not set
+        if: env.GITHUB_TOKEN == ''
+        run: |
+          echo "The `GITHUB_PAT` input is required."
```

Comment:
Is it `GITHUB_TOKEN` or `GITHUB_PAT`?

Reply:
There's some reserved name collisions:

- `GITHUB_TOKEN` cannot be used as a workflow input, so we call it `GITHUB_PAT` and compute the local env variable `GITHUB_TOKEN` using that. The variable is used implicitly in many actions.

- I just pushed another commit that defaults to `HUGRBOT_PAT` if the secret `GITHUB_PAT` is not defined. I tried to add a `GITHUB_PAT` secret to the repo, but apparently we're not allowed to define `GITHUB_*` secrets.
`HUGRBOT_PAT` is not an input to the workflow call, so it will only be defined when running the action locally